client/me/purchases/manage-purchase/payment-method-selector/tos-text.tsx

```diff
@@ -1,8 +1,13 @@
 import { localizeUrl } from '@automattic/i18n-utils';
 import { useTranslate } from 'i18n-calypso';
 import InlineSupportLink from 'calypso/components/inline-support-link';
+import type { FC } from 'react';
 
-export default function TosText() {
+interface TosTextProps {
+	isAkismetPurchase: boolean;
+}
+
+const TosText: FC< TosTextProps > = ( { isAkismetPurchase } ) => {
```

Comment:
Nitpick/ curious, any reason not to keep `export default function TosText()` and consume the props as arguments to the component function versus using the `FC` type here?

Reply:
The honest reason was I am most comfortable doing it that way, but your question did make me wonder exactly why I do it this way. I think this was the standard at one point but I am now finding some articles about how its [not best to use it in this way because it implies the `children` prop](https://juhanajauhiainen.com/posts/should-you-use-reactfc-to-type-your-components). I will go ahead and remove it

Reply:
Thanks for the article link, I learned something new as well! :)